src/NServiceBus.Core/Licensing/LicenseDetailsProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace NServiceBus.Features
```

Comment:
Nitpick: not that it matters much but `NServiceBus.Licensing` or something like it would make more sense as a namespace